pkgs/desktops/gnome/extensions/collisions.json

```diff
@@ -20,37 +20,209 @@
       "Hide_Activities@shay.shayel.org",
       "hide-activities-button@nmingori.gnome-shell-extensions.org"
     ],
+    "proxy-switcher": [
+      "ProxySwitcher@flannaghan.com",
+      "ProxySwitcher@flannaghan.com"
+    ],
     "shutdowntimer": [
       "ShutdownTimer@neumann",
       "ShutdownTimer@deminder"
     ],
+    "mailnag": [
+      "mailnag@pulb.github.com",
+      "mailnag@pulb.github.com"
+    ],
+    "ping-indicator": [
+      "ping_indicator@trifonovkv.gmail.com",
+      "ping_indicator@trifonovkv.gmail.com"
+    ],
+    "synaptic-button": [
+      "synaptic-button@fthx",
+      "synaptic-button@fthx",
+      "synaptic-button@fthx"
+    ],
     "bottompanel": [
       "bottompanel@tmoer93",
       "bottom-panel@sulincix"
     ],
-    "noannoyance": [
```

Comment:
This entry seems gone. What should happen to the records in `extensionRenames.nix`?